itests/util/src/main/java/org/apache/hadoop/hive/ql/parse/CoreParseNegative.java

```diff
@@ -50,9 +50,8 @@ public void beforeClass() {
     firstRun = true;
     try {
       String hadoopVer = cliConfig.getHadoopVersion();
-      qt = new QTestUtil((cliConfig.getResultsDir()), (cliConfig.getLogDir()), miniMR, null,
-          hadoopVer,
-       initScript, cleanupScript, false, false);
+      qt = new QTestUtil((cliConfig.getResultsDir()), (cliConfig.getLogDir()), miniMR,
+        null, hadoopVer, initScript, cleanupScript, false);
```

Comment:
To leave it in the same state as in the PR "null," should be moved up to the previous line

````
      qt = new QTestUtil((cliConfig.getResultsDir()), (cliConfig.getLogDir()), miniMR, null,
          hadoopVer, initScript, cleanupScript, false);
````